Tasks/AzureWebAppV1/package.json

```diff
@@ -20,7 +20,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azurermdeploycommon": "^3.208.0",
+    "azure-pipelines-tasks-azurermdeploycommon": "3.219.1",
```

Comment:
Publish azure-pipelines-tasks-azurermdeploycommon and update to 3.221.0

Reply:
Done